src/react_native/reanimated.cljs

```diff
@@ -17,12 +17,15 @@
                              SlideInUp
                              SlideOutUp
                              LinearTransition
+                             enableLayoutAnimations
                              runOnJS)]
             [reagent.core :as reagent]
             ["react-native-redash" :refer (withPause)]
             [react-native.flat-list :as rn-flat-list]
             [utils.worklets.core :as worklets.core]))
 
+(enableLayoutAnimations true)
```

Comment:
Nice 👍 
Thanks for solving it @Parveshdhull 

Why don't we upgrade to reanimated v3?

Reply:
Hi @ulisesmac,
Thank you very much for reviewing PR. Actually upgrade is currently [blocked](https://github.com/status-im/status-mobile/issues/15764).